src/components/QuickFilters/QuickFiltersBlock/QuickFiltersBlock.react.js

```diff
@@ -4,20 +4,26 @@ export class QuickFiltersBlock extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
-      name: this.props.name
+      name: this.props.name,
+      label: this.props.label,
     };
   }
 
   render() {
     var filters = this.props.filters;
     var buttons = Object.keys(filters).map(function(filter) {
-      return (
-        <QuickFiltersButton filters={filters[filter]} name={filter} blockName={this.state.name}/>
-      );
+      if (filter != "label") {
+        return (
+          <QuickFiltersButton filters={filters[filter]} name={filter} blockName={this.state.name}/>
+        );
+      }
     }, this);
     return (
-      <div className="btn-group quick-filters-block">
-        {buttons}
+      <div>
+        {this.props.label}
```

Comment:
It's styled in the ![other PR](https://github.com/jobready/neptune/pull/2006/files).